app/models/project.rb

```diff
@@ -1,10 +1,13 @@
 class Project < ActiveRecord::Base
+  before_save :set_github_repo
   belongs_to :organization
   acts_as_ordered_taggable
   acts_as_ordered_taggable_on :technologies, :causes
 
-  attr_accessible :organization_id, :name, :url, :github_repo, :description, :notes, :cause_list, :technology_list, :is_active, :is_approved
-  validates_presence_of :name, :github_repo
+  attr_accessor :git_url
+  attr_accessible :organization_id, :name, :url, :github_repo, :description, :notes, :cause_list, :technology_list, :is_active, :is_approved, :git_url
+  validates_presence_of :name, :git_url
```

Comment:
I like `:submitted_github_url` here -- thanks for the great contribution, @sai92messy and the code suggestions, @courte!